### PR TITLE
Merge hvcfs cli

### PR DESCRIPTION
**Pull request by tcasstevens**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: src/test/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfsTest.kt [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+import com.github.ajalt.clikt.testing.test
+import net.maizegenetics.phgv2.utils.getChecksum
+import org.apache.logging.log4j.LogManager
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import java.io.File
+import kotlin.test.assertEquals
+
```
Comment (lynnjo) [inline src/test/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfsTest.kt:11]: If you add test cases that run the clikt command without the required parameters you should get additional code coveragte.

**Review by lynnjo — APPROVED** [review]

File: src/test/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfsTest.kt (hunk 1 above)
Comment (lynnjo) [reply src/test/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfsTest.kt:11]: oh - maybe not as you don't have a validate clause.  Coverage will be the same.

File: src/main/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfs.kt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    val outputFile by option(help = "Full path to output HVCF file")
+        .required()
+
+    val rangeBedfile by option(help = "Full path to range bedfile")
+
+    override fun run() {
+
+        val inputFiles = File(inputDir)
+            .walk()
+            .filter { it.isFile && (it.name.endsWith(".h.vcf") || (it.name.endsWith(".h.vcf.gz"))) }
```
Comment (aberthel) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfs.kt:41]: load-vcf also allows ".hvcf" and ".hvcf.gz". We should keep accepted file extensions consistent between commands.

**Review by aberthel — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfs.kt (hunk 2 above)
Comment (tcasstevens) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/MergeHvcfs.kt]: Yes, thank you! I added those

**Review by pjbradbury — APPROVED** [review]

**Pull request merged**